### PR TITLE
Updates call_number_type element based on Holdings Record

### LINKED ISSUE #6: Spinomatic XML not returning correct call number type
SUDOC call numbers are being returned with <call_number_type_name>Library of Congress classification</call_number_type_name> and therefore not being formatted correctly. Collection Care accidentally sent out some incorrectly formatted call numbers as a result.

e.g. 36105235065898
Per Jeremy - looking at XSLT stylesheet that Setae uses, the call_number_type is hardcoded to be LC: https://github.com/sul-dlss/folio-setae-api/blob/ae9ad0e9779e0e0363745899e6687270350dbf6e/app/alma-rest-item.xsl#L47
alma-rest-item.xsl
            call_number_type desc="Library of Congress classification"0/call_number_type
<https://github.com/sul-dlss/folio-setae-api|sul-dlss/folio-setae-api>sul-dlss/folio-setae-api | Added by GitHub

Maybe we could ask the setae developer at Cornell for insight here?

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #6

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -242,6 +250,56 @@ def _retrieve_permanent_location(holdings_id: str, okapi_headers: dict) -> tuple
...
+    """
+    call_number_uuid = holdings['callNumberTypeId']
+    call_number_type_result = requests.get(
+        f"{os.getenv('OKAPI_URL')}/call-number-types/{call_number_uuid}",
+        headers=okapi_headers
+    )
+    call_number_type_result.raise_for_status()
+    name = call_number_type_result.json()["name"]
+    code = None
+    # Matches Ex Libris codes and names at https://developers.exlibrisgroup.com/alma/apis/docs/xsd/rest_item.xsd/?tags=GET#holding_data
```
Comment (jermnelson) [inline app/main.py:265]: @ahafele - the following names and codes are from the Ex Libris link, does this align with the what the users are expecting?

**Comment by ahafele** [comment]
@jermnelson just checking in on this work...any ideas on timeline?

**Review by jgreben — APPROVED** [review]

**Comment by ahafele** [comment]
@jermnelson or @jgreben was this put into production? I can see the change when connecting to folio-setae-api-test.stanford.edu but on prod it is the old result.

**Comment by jgreben** [comment]
I think @jermnelson put this into production.

**Comment by ahafele** [comment]
Jeremy updated something and it is now working in prod

**Pull request merged**